### ISSUE TITLE
Label is not update

### ISSUE THREAD

**Issue by SaundersDong** [alibaba/BizCharts#184]
<html>
<head>
    <script src="https://unpkg.com/react@15.6.2/dist/react.min.js"></script> <script src="https://unpkg.com/react-dom@15.6.2/dist/react-dom.min.js"></script> <script src="https://cdn.bootcss.com/babel-core/5.8.38/browser.min.js"></script> <script type="text/javascript" src="//gw.alipayobjects.com/os/antv/assets/data-set/0.8.3/data-set.min.js"></script> <script src="https://unpkg.com/bizcharts@3.1.0/umd/BizCharts.min.js"> </script> <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
  </head>
  <body>
      <div id="mountNode"><div data-reactroot=""><button>1</button><button>2</button><div><div><div style="position:relative;"><canvas id="canvas_1" width="1102" height="800" style="width: 551px; height: 400px;"></canvas><div class="g2-tooltip" style="position: absolute; visibility: hidden; white-space: nowrap; z-index: 8; transition: visibility 0.2s cubic-bezier(0.23, 1, 0.32, 1), left 0.4s cubic-bezier(0.23, 1, 0.32, 1), top 0.4s cubic-bezier(0.23, 1, 0.32, 1); background-color: rgba(0, 0, 0, 0.65); border-radius: 4px; color: rgb(255, 255, 255); font-size: 12px; font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, &quot;Helvetica Neue&quot;, Helvetica, &quot;PingFang SC&quot;, &quot;Hiragino Sans GB&quot;, &quot;Microsoft YaHei&quot;, SimSun, sans-serif; line-height: 20px; padding: 10px 10px 6px;"><div class="g2-tooltip-title"></div><ul class="g2-tooltip-list"></ul></div></div></div></div></div></div>
      <script type="text/babel">
       
        var ViewChart = React.createClass({
			getInitialState: function() {	
				return {btnName: 1};
			},
				click: function(btnName) {
					this.setState({btnName : btnName})
				},
				render: function() {
					var btnName = this.state.btnName;
					const { Chart, Axis, Geom,Label, Tooltip, Legend } = window.BizCharts;
					const data_1 = [
						{ name:'London', 'Jan.': 18.9, 'Feb.': 28.8, 'Mar.' :39.3, 'Apr.': 81.4, 'May': 47, 'Jun.': 20.3, 'Jul.': 24, 'Aug.': 35.6 },
						{ name:'Berlin', 'Jan.': 12.4, 'Feb.': 23.2, 'Mar.' :34.5, 'Apr.': 99.7, 'May': 52.6, 'Jun.': 35.5, 'Jul.': 37.4, 'Aug.': 42.4}
					];
					const data_2 = [
						{ name:'China', 'Jan.': 20.9, 'Feb.': 30.8, 'Mar.' :19.3, 'Apr.': 81.4, 'May': 47, 'Jun.': 21.3, 'Jul.': 44, 'Aug.': 35.6 },
						{ name:'US', 'Jan.': 11.4, 'Feb.': 2.2, 'Mar.' :4.5, 'Apr.': 49.7, 'May': 22.6, 'Jun.': 30.5, 'Jul.': 37.4, 'Aug.': 42.4}
					];
					const ds = new DataSet();
					const dv = ds.createView().source(btnName == 1 ? data_1 : data_2);
					dv.transform({
						type: 'fold',
						fields: [ 'Jan.','Feb.','Mar.','Apr.','May','Jun.','Jul.','Aug.' ], // 展开字段集
						key: '月份', // key字段
						value: '月均降雨量', // value字段
					});
					return <div>
									<button onClick={this.click.bind(this,1)} >1</button> 
									<button onClick={this.click.bind(this,2)} >2</button> 
									<Chart height={400} data={dv} forceFit>
									<Axis name="月份" />
									<Axis name="月均降雨量" />
									<Legend />
									<Tooltip crosshairs={{type : "y"}}/>
										<Geom type={btnName == 1 ? 'interval' : 'line'} position="月份*月均降雨量" color={'name'} adjust={[{type: 'dodge',marginRatio: 1/32}]} >
                                      <Label content = '月均降雨量' />
                                      </Geom>
								</Chart>
								 </div>;
				}
			});
			ReactDOM.render(
			<ViewChart />,
			document.getElementById('mountNode')
			);
    </script>
</body>
</html>

第一次Label可以显示文本, 但当我来回切换的时候, 此时Label就消失了, 如何解决呢?

**Comment by weepy3641** [comment]
1、暂时可以用如下方案解决此问题：
```jsx
<Geom type={btnName == 1 ? 'interval' : 'line'} position="月份*月均降雨量" color={'name'} adjust={[{type: 'dodge',marginRatio: 1/32}]} >
                                  <Label content = '月均降雨量' />

btnName == 1 ? 
  <Geom type='interval' position="月份*月均降雨量" color={'name'} adjust={[{type: 'dodge',marginRatio: 1/32}]} >
    <Label content = '月均降雨量' /> 
  </Geom>
  :
  <Geom type='line' position="月份*月均降雨量" color={'name'} adjust={[{type: 'dodge',marginRatio: 1/32}]} >
    <Label content = '月均降雨量' /> 
  </Geom>
```

2、彻底修复需要等下个版本。

**Comment by weepy3641** [comment]
@SaundersDong 
fix at 3.1.3
